doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1065,11 +1065,12 @@ use cases of this functionality is the association of Context Providers.
 
 A context registration is represented by a JSON object with the following fields:
 
-+ `id`: Unique identifier assigned to the registration. Automatically created at creation time.
-+ `subject` : It s an object that describes the subject of the registration.
-+ `callback` : URL pointing to the service which is registered. In the case of a Context Provider
-corresponds to the URL of the provider service.
-+ `duration`: Duration of the registration in ISO8601 format. Default duration is infinite.
++ `id`       : Unique identifier assigned to the registration. Automatically generated at creation time.
++ `subject`  : It s an object that describes the subject of the registration.
++ `callback` : URL which denotes the end point of the registered service. It can be undefined under certain conditions,
+for instance, transient unavailability of the provider.
++ `metadata` : An optional dictionary which allows to associate extra metadata to the registration. 
```

Comment:
As far as I remember, registration metadata are of the same type that entity attribute metadata, so they have a name, type and value. Thus, description should be aligned with the one at https://github.com/telefonicaid/fiware-orion/pull/1361:

> metadata: a JSON object which represents attribute metadata. This property is optional in requests, meaning that the attribute doesn't have metadata. In responses, this property is set to `{}` if the attribute doesn't have metadata. Each property in this object represents a metadata element (which name is the name of the property) which value is
a JSON object composed of two properties: `value` and `type` (`value` and `type` following the same rules that the
ones described in the previous two bullets).

Probably is a good idea to merge first PR #1361, then the present PR, to ensure that both parts of the .apib are coherent.